src/OpenLoco/src/GameCommands/GameCommands.h

```diff
@@ -343,7 +343,7 @@ namespace OpenLoco::GameCommands
             , rotation(regs.bh & 0x3)
             , trackId(regs.dl & 0x3F)
             , index(regs.dh)
-            , trackObjectId(regs.ebp)
+            , trackObjectId(static_cast<uint8_t>(regs.ebp)) // TODO: Validate this.
```

Comment:
This is safe

Reply:
I know this is safe but why use the 32 bit width of that register? That's why I added the validate comments all over as I'm not sure if thats the actual register used and we could avoid the casts if we just use the actual 8 bit version of it.

Reply:
There is no 8bit ebp register. Conversion from registers to values it doesn't matter much if the register is wider than it needs to be. The other way round though should be investigated to ensure its correct.